python/servo/package_commands.py

```diff
@@ -96,8 +96,11 @@ def copy_dependencies(binary_path, lib_path):
 
 def copy_windows_dependencies(binary_path, destination):
     deps = [
+        "api-ms-win-crt-runtime-l1-1-0.dll",
         "libcryptoMD.dll",
         "libsslMD.dll",
+        "msvcp140.dll",
+        "vcruntime140.dll",
```

Comment:
Couple questions - 

Is this better than installing the redist as part of the Servo install?  We can bundle some files, but was wondering if it'd be better to just have the redist get installed when Servo is installed.

Second, where does "api-ms-win-crt-runtime-l1-1-0.dll" come from (and do we need it)?  Poking around https://www.visualstudio.com/en-us/productinfo/2017-redistribution-vs I don't see it as an allowed redist file.